test/functional/eval/input_spec.lua

```diff
@@ -196,6 +228,19 @@ describe('input()', function()
     eq('Vim(call):E118: Too many arguments for function: input',
        exc_exec('call input("prompt> ", "default", "file", "extra")'))
   end)
+  it('supports highlighting', function()
+    feed([[:call input({"highlight": "RainBowParens"})<CR>]])
+    wait()
+    feed('(())')
+    wait()
```

Comment:
@bfredl In the test where I observed calling that twice I had 118 columns, no prompt and one to four characters (typed `(())` with a slightly adjusted RainBowParens function). In the input_spec while screen is rather narrow it still fits.